RNA_CCLE.ipynb

```diff
@@ -31,58 +29,948 @@
     "%autoreload 2\n",
     "\n",
     "output_notebook()"
-   ]
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stderr",
+     "text": [
+      "/Users/xiaomeng/miniconda3/lib/python3.9/site-packages/rpy2/robjects/pandas2ri.py:14: FutureWarning: pandas.core.index is deprecated and will be removed in a future version.  The public classes are available in the top-level namespace.\n",
```

Comment:
you should drop the outputs from notebooks before merging... you can use nbstripout for that..

Reply:
resolved in new PR